Integrations/MailSenderNew/MailSenderNew.py

```diff
@@ -286,6 +287,10 @@ def create_msg():
         msg['To'] = header(','.join(to))
     if cc:
         msg['CC'] = header(','.join(cc))
+    if additional_header:
+        for h in additional_header:
+            header_name_value = h.split('=')
+            msg.add_header(header_name_value[0], header(header_name_value[1]))
```

Comment:
Your changes weren't working for me. I think you have a small bug here. I think it should be 
```
msg.add_header(header_name_value[0], header_name_value[1])
```

Reply:
You are correct. I wanted to take advantage of the `header` function which is able to handle new line characters correctly. But unfortunately it returns a `Header` object which is not supposed to be added as a parameter here. 
I'll update the code accordingly.

Reply:
@avidan-H Thanks for your hint. PR is updated to add the headers using the `msg` dict directly instead of the `add_header` function (like it's done for the existing headers also).

Reply:
awesome 👍